Simulation/Headers/Components/com_Physics.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <glm/glm.hpp>
+
+#include "ECS/Types.h"
+
+#include "Algorithms/Physics.h"
+
+#include "Components/com_Transform.h"
+
+namespace Component
+{
+	class com_Physics
+	{
+	public:
+		com_Physics() {};
+		com_Physics(float mass, float invmass, float res, glm::vec3 com, glm::mat4 inertia);
```

Comment:
constructor needs: int width, height, depth to calculate com. inertia will be based of the box model, so wont need passing in either